CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+1.13.0 (in progress)
+====================
+
+New Features
+------------
+
+- Previously, if a Room was completed via the REST API while your
+  twilio-video.js-based application was connected to it, the Room would emit a
+  "disconnected" event, but it wouldn't explain _why_ the disconnect occurred.
+  Now, the "disconnected" event will include a TwilioError 53118, "Room
+  completed" (JSDK-1884). In some applications, this will be expected, so you
+  should set an event listener on the Room as follows:
+
+  ```js
+  room.once('disconnected', (room, error) => {
+    switch (error.code) {
```

Comment:
@markandrus 

What about the case where there is no `error`?

Reply:
Good point